nal_eval/tests/fixtures/branch.nal

```diff
@@ -0,0 +1,111 @@
+if true {
+    print(10)
+}
+
+if false {
+    print(-10)
+}
+
+if true {
+    print(20)
+} else {
+    print(-20)
+}
+
+if false {
+    print(-30)
+} else {
+    print(30)
+}
+
+if true {
+    print(40)
+} else if true {
```

Comment:
Though I have strong intention to add `else if` syntax in near future, it's not yet implemented. So please remove them for now.
You can just comment them out, and uncomment them when it's implemented.